### PR TITLE
Issues/362/accumulo 1.8

### PR DESCRIPTION
**Pull request by vincentrussell**
Please take a look at this to make sure I didn't screw anything up.

Thanks,

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,14 +31,17 @@ limitations under the License.
   <description>The Accumulo Recipes project aims to provide reusable services for architectures utilizing Apache
         Accumulo.</description>
   <modules>
+    <module>minicluster-test-driver</module>
```
Comment (eawagner) [inline pom.xml]: Why not include this in the test module?

File: pom.xml [hunk 2]
```diff
@@ -317,6 +340,129 @@ limitations under the License.
...
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>build-helper-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>reserve-tomcat-port</id>
```
Comment (eawagner) [inline pom.xml]: should probably change the name of the id for this.

File: pom.xml [hunk 3]
```diff
@@ -317,6 +340,129 @@ limitations under the License.
...
+                  <includes>
+                    <include>**/*IT.java</include>
+                  </includes>
+                  <skipTests>${skipTests}</skipTests>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>com.google.code.sortpom</groupId>
```
Comment (eawagner) [inline pom.xml]: Why do we have sort pom in the profile. Isn't this already (or should be) done outside of a profile?

File: store/entity-store/src/main/java/org/calrissian/accumulorecipes/entitystore/hadoop/EntityOutputFomat.java [hunk 4]
```diff
@@ -20,7 +20,6 @@
 import org.apache.accumulo.core.client.*;
 import org.apache.accumulo.core.client.mapreduce.lib.impl.OutputConfigurator;
 import org.apache.accumulo.core.client.security.tokens.AuthenticationToken;
-import org.apache.accumulo.core.util.ArgumentChecker;
```
Comment (eawagner) [inline store/entity-store/src/main/java/org/calrissian/accumulorecipes/entitystore/hadoop/EntityOutputFomat.java]: Thank you, this is one of my pet peeves.

**Comment by eawagner** [comment]
Other than a few small issues the gist of the MR is good. Mostly, just some cleanup stuff.

File: commons/src/main/java/org/calrissian/accumulorecipes/commons/iterators/support/NodeToJexl.java [hunk 5]
```diff
@@ -20,9 +20,9 @@
 
 import java.util.*;
 
+import static org.apache.commons.lang.StringUtils.contains;
```
Comment (vincentrussell) [inline commons/src/main/java/org/calrissian/accumulorecipes/commons/iterators/support/NodeToJexl.java]: commons-lang3 isn't on the classpath.    I didn't feel like I should ad it.

File: pom.xml (hunk 3 above)
Comment (vincentrussell) [inline pom.xml]: not sure how that happened.

File: pom.xml (hunk 1 above)
Comment (vincentrussell) [inline pom.xml]: Oh... i think my reason was that I didn't want to add a test dependency on the mini cluster if there wasn't a need, but maybe that was overkill.

**Pull request merged**